index.js

```diff
@@ -1,14 +1,17 @@
 var less = require('less');
 var sysPath = require('path');
 var progeny = require('progeny');
+var _ = require('underscore');
 
 function LESSCompiler(config) {
   if (config == null) config = {};
   if (config.plugins == null) config.plugins = {};
 
-  this.config = config.plugins.less || {};
+  this.options = _.extend({}, config.plugins.less);
```

Comment:
And why bother with that? Won't clean-css or whatever optimizer just strip out the comments if they are not desired?